package.json

```diff
@@ -3,27 +3,33 @@
   "version": "4.0.3",
   "description": "`ObservableStore` is a synchronous in-memory store for a single value, that you can subscribe to updates on.",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "files": [
-    "lib/*.js"
+    "lib/*.js",
+    "lib/*.d.ts"
   ],
   "scripts": {
-    "test": "tape test/**/*.js",
-    "lint": "eslint .",
+    "build": "tsc --project .",
+    "test": "yarn build && tape test/**/*.js",
+    "lint": "yarn build && eslint test/*.js",
```

Comment:
It looks like the `.ts` files aren't being linted? :thinking:

Reply:
They aren't, no. That's a separate PR.

Reply:
I wanted to make minimal changes to the JS source to facilitate review. Linting the TS sources would add a lot of changes/noise to the diff.